java.jdbc-ballerina/build.gradle

```diff
@@ -265,135 +269,130 @@ task initializeVariables {
+                if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                    commandLine 'cmd', '/c', "$distributionBinPath/bal.bat doc && exit %%ERRORLEVEL%%"
+                } else {
+                    commandLine 'sh', '-c', "$distributionBinPath/bal doc"
+                }
             }
-        }
-        copy {
-            from file("$project.projectDir/target/apidocs/${packageName}")
-            into file("$project.projectDir/build/docs_parent/docs/${packageName}")
-        }
-    }
-
-    outputs.dir artifactCacheParent
-    outputs.dir artifactBallerinaDocs
-    outputs.dir artifactLibParent
-}
-
-task ballerinaPublish {
-
-    ext.balPush = { additionalParams ->
-        exec {
-            workingDir project.projectDir
-            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push ${additionalParams} && exit %%ERRORLEVEL%%"
-            } else {
-                commandLine 'sh', '-c', "$distributionBinPath/bal push ${additionalParams} "
+            copy {
+                from file("$project.projectDir/target/apidocs/${packageName}")
+                into file("$project.projectDir/build/docs_parent/docs/${packageName}")
             }
         }
-    }
-
-    doLast {
-        // Publish to local repository
-        def repository = ""
-        if (project.hasProperty("repository")) {
-            repository = project.findProperty("repository")
-        }
 
-        if (repository == "local") {
-            println("Publishing to the ballerina local repository...")
-            balPush("--repository=local")
-        } else if (repository == "central") {
-            // Publish to central
-            println("Publishing to the ballerina central..")
+        if (needPublishToCentral) {
             if (project.version.endsWith('-SNAPSHOT')) {
                 throw new InvalidUserDataException("Cannot push SNAPSHOT version to central")
             }
             if (ballerinaCentralAccessToken != null) {
-                balPush("")
+                println("Publishing to the ballerina central..")
+                exec {
+                    workingDir project.projectDir
+                    environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+                    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                        commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push && exit %%ERRORLEVEL%%"
+                    } else {
+                        commandLine 'sh', '-c', "$distributionBinPath/bal push"
+                    }
+                }
             } else {
                 throw new InvalidUserDataException("Central Access Token is Not Present")
             }
-        } else {
-            println("Publish task does not publish to ballerina repository")
+        }
+
+        if (needPublishToLocalCentral) {
```

Comment:
Should be else if https://github.com/ballerina-platform/module-ballerina-time/pull/100

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/130/commits/2c924a74de8688cb4ef635170dc801d935323a8a